### PR TITLE
Rewrite test script to python. Invoke it from .travis.yml. Add some docs.

### PR DESCRIPTION
**Pull request by franekp**

### CODE REVIEW

File: run_tests.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        shell=True,
+    )
+
+def run_python(args):
+    run_python_tests(coverage=args.coverage)
+    if not args.no_lint:
+        run_python_linter()
+
+def run_javascript_tests(coverage=False):
+    print('TODO: js tests')
```
Comment (berggren) [inline run_tests.py]: return NotImplemented instead

File: run_tests.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+def run_javascript_linter():
+    print('TODO: js linter')
```
Comment (berggren) [inline run_tests.py]: return NotImplemented instead

File: run_tests.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+def run_javascript(args):
+    run_javascript_tests(coverage=args.coverage)
+    if not args.no_lint:
+        run_javascript_linter()
+
+def run_selenium(args):
+    print('TODO: selenium tests')
```
Comment (berggren) [inline run_tests.py]: return NotImplemented instead

File: run_tests.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                shell=True,
+            )
+        else:
+            subprocess.check_call(['nosetests'])
+    finally:
+        subprocess.check_call(['rm', '-f', '.coverage'])
+
+def run_python_linter():
+    """Run pylint on timesketch and api_client."""
+    subprocess.check_call(['pylint', 'timesketch'])
```
Comment (berggren) [inline run_tests.py]: Will this use our utils/pylintrc?

File: run_tests.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        + ' pylint timesketch_api_client',
+        shell=True,
+    )
+
+def run_python(args):
+    run_python_tests(coverage=args.coverage)
+    if not args.no_lint:
+        run_python_linter()
+
+def run_javascript_tests(coverage=False):
```
[the remark this answers is not in the document]
Comment (berggren) [reply run_tests.py]: remove the arg?

File: run_tests.py (hunk 4 above)
Comment (franekp) [reply run_tests.py]: Yes, because utils/pylintrc was moved to <project_root>/.pylintrc

Comment (berggren) [reply run_tests.py]: Great!

**Comment by berggren** [comment]
LGTM

**Pull request merged**